src/gsy_e/models/area/__init__.py

```diff
@@ -252,6 +252,18 @@ def get_results_dict(self):
         return {}
 
 
+class CoefficientAreaException(Exception):
+    """Exception that is raised when serializing an area."""
+
+
+def assert_coefficient_area_settings(
+        strategy: Optional[SCMStrategy], setting: Optional[float], setting_name: str) -> float:
+    """Check if coefficient area that is not an asset provided SCM setting."""
+    if strategy is None and setting is None:
```

Comment:
I think that this would still fail ATM because of 2 areas: the "Community" area (which should be there) and the "Grid" area (which should not be there, but the UI is probably in the process of removing this). Therefore I would suggest to pass the area instead and check if it is a home area (essentially if all the children of this area have non-none strategy members). Thanks!

Reply:
I thought this check also applies to the Grid and community area. But you are right, all the settings are only used in the SCM homes. Will change this. Thanks!

Reply:
[a6cadd9](https://github.com/gridsingularity/gsy-e/pull/1544/commits/a6cadd97b72f56f44f60b5367179d6460cd9a882)